packages/fhir-health-care-service/src/components/HealthCareAddEdit/tests/index.test.tsx

```diff
@@ -0,0 +1,182 @@
+/* eslint-disable @typescript-eslint/camelcase */
```

Comment:
Is there a test suite that actually checks that a user can edit a health care resource?

Reply:
there is https://github.com/opensrp/web/pull/685/files#diff-a660fa6e2215952db8e273b28548b4cf4cd33d1d9d750d81ea483b379c704c8eR155-R174

Reply:
@ashfahan I was thinking more in line with functional test principles, akin to how the user would actually edit a health care resource using the ui

Reply:
@ashfahan IMO this is an integral part of the code being added, we do not want to collect too much tech debt where avoidable, please work on this and this will be ready to ship.